### PR TITLE
chore: simplify bug template

### PR DESCRIPTION
**Pull request by rarkins**
## Changes:

Simplifies existing bug template

## Context:

I noticed there was duplicate information requested

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added unit tests, or
- [ ] No new tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -9,31 +9,18 @@ labels: 'type:bug, status:requirements, priority-5-triage'
       Email renovate-disclosure@whitesourcesoftware.com instead.
 -->
 
-**What Renovate type, platform and version are you using?**
-
-<!-- Tell us if you're using the hosted App, or if you are self-hosting Renovate yourself. Platform too (GitHub, GitLab, etc) plus which version of Renovate if you're self-hosted. -->
-
```
Comment (HonkingGoose) [inline .github/ISSUE_TEMPLATE/bug_report.md:15]: The comment here can be removed, I agree!

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -9,31 +9,18 @@ labels: 'type:bug, status:requirements, priority-5-triage'
...
-**Renovate platform**
-
-- [ ] Azure DevOps (dev.azure.com)
-- [ ] Azure DevOps Server (self-hosted)
-- [ ] Bitbucket Cloud (bitbucket.org)
-- [ ] Bitbucket Server (self-hosted)
-- [ ] Gitea (gitea.com, self-hosted)
-- [ ] GitHub (github.com, GitHub Enterprise)
-- [ ] GitLab (gitlab.com, self-hosted)
```
Comment (HonkingGoose) [inline .github/ISSUE_TEMPLATE/bug_report.md:29]: I see you've removed the platform checkboxes (Azure, Bitbucket, etc.), what is the reason that you're removing those?

My reasoning to have the checkboxes is to make it easy to find the supported platforms, and the ways in which that platform can be used.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -9,31 +9,18 @@ labels: 'type:bug, status:requirements, priority-5-triage'
...
+- [ ] Self hosted:
 
-**Renovate version (if self hosting)**
+Platform: ...
 
-I'm self hosting Renovate with version: ...
+Renovate version: ...
```
Comment (HonkingGoose) [inline .github/ISSUE_TEMPLATE/bug_report.md:19]: So we also need to know the Renovate version if people are using the hosted app?

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (rarkins) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Added back

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (rarkins) [reply .github/ISSUE_TEMPLATE/bug_report.md:32]: No

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -9,32 +9,31 @@ labels: 'type:bug, status:requirements, priority-5-triage'
...
-- [ ] GitHub (github.com, GitHub Enterprise)
-- [ ] GitLab (gitlab.com, self-hosted)
-
-**Renovate version (if self hosting)**
+- [ ] Bitbucket Server
+- [ ] Gitea
+- [ ] github.com
+- [ ] GitHub Enterprise Server
+- [ ] gitlab.com
+- [ ] GitLab self-hotsed
```
Comment (HonkingGoose) [inline .github/ISSUE_TEMPLATE/bug_report.md:30]:
```suggestion
- [ ] GitLab self-hosted
```

A typo slipped in. 😉

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -65,4 +64,4 @@ Please read the [minimal reproductions documentation](https://github.com/renovat
 
 **Additional context**
 
-<!-- Add any other context about the problem here, including your own debugging or ideas on what went wrong. -->
+...<!-- Add any other context about the problem here, including your own debugging or ideas on what went wrong. -->
```
Comment (HonkingGoose) [inline .github/ISSUE_TEMPLATE/bug_report.md:67]:
```suggestion
<!-- Add any other context about the problem here, including your own debugging or ideas on what went wrong. -->
...
```

I think the comment and the `...` can be swapped around?

**Review by HonkingGoose — CHANGES_REQUESTED** [review]
Some small fixes.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 6]
```diff
@@ -66,3 +65,4 @@ Please read the [minimal reproductions documentation](https://github.com/renovat
 **Additional context**
 
 <!-- Add any other context about the problem here, including your own debugging or ideas on what went wrong. -->
+...
```
Comment (HonkingGoose) [inline .github/ISSUE_TEMPLATE/bug_report.md:68]:
```suggestion

...
```

Add newline to resolve the linter fail. 😄

**Review by HonkingGoose — APPROVED** [review]

**Comment by HonkingGoose** [comment]
@rarkins This won't auto-merge because its waiting for an approving review by a reviewer with **write** level access. My approving review is only at **triage** level access. 😄

**Pull request merged**